guides/common/modules/con_postgresql-tuning.adoc

```diff
@@ -5,6 +5,9 @@ PostgreSQL is the primary SQL based database that is used by {Project} for the s
 The database sees an extensive usage is usually working on to provide the {Project} with the data which it needs for its smooth functioning.
 This makes PostgreSQL a heavily used process which if tuned can have a number of benefits on the overall operational response of {Project}.
 
+The PostgreSQL authors recommend disabling Transparent Huge Pages on server running PostgreSQL.
```

Comment:
```suggestion
The PostgreSQL authors recommend disabling Transparent Huge Pages on servers running PostgreSQL.
```

Reply:
Oh, got it. "PostgreSQL authors recommend it on **servers**", "We are running it on **server**" :)

Reply:
Well, I think it should be either `on a/the server` or `on servers`. And I think `on the servers` is also valid, but I'd not use that here.